cmd/image/qcow2ova/prep/templates.go

```diff
@@ -49,6 +47,16 @@ yum-config-manager --disable Advance_Toolchain
 yum-config-manager --add-repo=https://public.dhe.ibm.com/software/server/POWER/Linux/yum/IBM/RHEL/$(rpm -E %{rhel})/ppc64le/
 rpm --import https://public.dhe.ibm.com/software/server/POWER/Linux/yum/IBM/RHEL/$(rpm -E %{rhel})/ppc64le/repodata/repomd.xml.key
 {{end}}
+{{if eq .Dist "fedora"}}
+sed -i 's/^less \/opt\/ibm\/lop\/notice/#less \/opt\/ibm\/lop\/notice/g' /opt/ibm/lop/configure
+yum-config-manager --add-repo=https://public.dhe.ibm.com/software/server/POWER/Linux/yum/IBM/Fedora/ppc64le/
+rpm --import https://public.dhe.ibm.com/software/server/POWER/Linux/yum/IBM/Fedora/ppc64le/repodata/repomd.xml.key
+{{else}}
+sed -i 's/^more \/opt\/ibm\/lop\/notice/#more \/opt\/ibm\/lop\/notice/g' /opt/ibm/lop/configure
+{{end}}
+ 
+echo 'y' | /opt/ibm/lop/configure
+
```

Comment:
part of this is already in the template , only repo changes

Reply:
`$(rpm -E %{rhel})` rpm macro are getting used for rhel and centos distros, but I am not able to use it in the fedora, hence I have it as separate block for fedora